build.py

```diff
@@ -61,7 +65,26 @@ def build_webgl_game(gamedir, output_dir):
       if os.path.isfile(gamedir):
             shutil.copyfile(gamedir, output_dir + love_game_name)
       else:
-            shutil.make_archive(output_dir + love_game_name, 'zip', gamedir)
+            # Original: shutil.make_archive(output_dir + love_game_name, 'zip', gamedir)
+            # Revised: make_archive should skip hidden files and directories, like .git, etc. 
+            try:
+                  # Copy all non-hidden files and directories from gamedir to temp_dir
+                  for item in os.listdir(gamedir):
+                        if item.startswith('.'):
+                              continue
+                        src_path = os.path.join(gamedir, item)
+                        dest_path = os.path.join(temp_dir, item)
+                        if os.path.isdir(src_path):
+                              shutil.copytree(src_path, dest_path, ignore=shutil.ignore_patterns('.*'))
+                        else:
+                              shutil.copy2(src_path, dest_path)
+                  
+            # Create the archive from the temporary directory
+                  shutil.make_archive(os.path.join(output_dir, love_game_name), 'zip', temp_dir)
```

Comment:
Can we just use copytree instead of the loop?
```suggestion
                  temp_game_dir = os.path.join(temp_dir, 'game')
                  shutil.copytree(gamedir, temp_game_dir, ignore=shutil.ignore_patterns('.*'))

            # Create the archive from the temporary directory
                  shutil.make_archive(os.path.join(output_dir, love_game_name), 'zip', temp_game_dir)
```